packages/playground/remote/src/lib/boot-playground-remote.ts

```diff
@@ -230,6 +230,20 @@ function setupPostMessageRelay(
 
 		window.parent.postMessage(event.data, '*');
 	});
+
+	window.addEventListener('message', (event) => {
+		if (event.source !== window.parent) {
+			return;
+		}
+
+		if (typeof event.data !== 'object' || event.data.type !== 'relay') {
+			return;
+		}
+
+		console.log("passing message to iframe's window", event);
+
+		wpFrame?.contentWindow?.postMessage(event.data, event.origin);
```

Comment:
`event.origin` is the source origin, but `postMessage` expects target origin as its second argument. You could just omit that second argument – it's optional and we are certain we have the right frame here.